cmd/branchProtection_i_test.go

```diff
@@ -32,6 +44,100 @@ func (suite *BranchProtectionSuite) TestCreateBranchProtection() {
 	suite.Assert().Contains(protection.RequiredStatusChecks.Contexts, "linkChecker")
 }
 
+func (suite *BranchProtectionSuite) TestBranchProtectionMissing() {
+	suite.cleanup()
+	defer suite.cleanup()
+	output := captureOutput(func() {
+		branchProtectionCmd.Run(branchProtectionCmd, []string{testRepo})
+	})
+	suite.Assert().Equal("exasol/testing-release-robot does not have a branch protection rule for default branch master. Use --fix to create it. This error can also happen if you don't have admin privileges on the repo.", output)
```

Comment:
not sure, since it's an internal tool.
@redcatbear ?

Reply:
It's not strictly necessary for internal tools. Does not hurt though either, since the tags separate the projects and if you don't search for this, you will not find it. I also don't see it as a big deal should that appear in the catalog.

That being said. Keep it simple. If we don't need it to be in the catalog, why invest the effort?